audbackend/core/backend.py

```diff
@@ -286,73 +308,6 @@ def ls(
         """
         return sorted(self._ls(path))
 
-    def _path(
-            self,
-            folder: str,
-            name: str,
-            ext: str,
-            version: str,
-    ) -> str:  # pragma: no cover
-        r"""File path on backend."""
-        raise NotImplementedError()
```

Comment:
In the single backend you still implement `_path()`.
But I guess with removing it from `Backend` you indicate that a developer don't has to implement it, correct.
Which means inside `Artifactory` and `Filesystem` you just make it out of convenience?

Reply:
yes